go.mod

```diff
@@ -476,12 +470,15 @@ require github.com/grafana/grafana/apps/alerting/notifications v0.0.0-2024120916
 
 require github.com/grafana/grafana/apps/investigation v0.0.0-20241218083103-f46c07aba7b6 // @fcjack @matryer
 
+require github.com/grafana/grafana/pkg/plugins/codegen v0.0.0-00010101000000-000000000000 // @grafana/grafana-as-code
```

Comment:
Does the `@grafana/grafana-as-code` team still exist? Side question: what owner would actually make sense for these things?

Reply:
AFAIK there are some mentions to this squad, but I not really sure which people are on it...